packages/babel-plugin-emotion/test/util.js

```diff
@@ -25,38 +26,43 @@ jest.mock('fs')
 fs.existsSync.mockReturnValue(true)
 fs.statSync.mockReturnValue({ isFile: () => false })
 
-const createInlineTester = transform => opts => {
+const isBabel7 = babel => parseInt(babel.version.split('.')[0], 10) === 7
```

Comment:
babel@6 duplicates some nodes, so we just want to validate this when using babel@7 to avoid false positives causes by babel itself